app/jobs/create_rss_notifications.rb

```diff
@@ -5,7 +5,7 @@ class CreateRssNotifications < ApplicationJob
 
   def perform(options = {})
     # Defaults to theforeman.org blog RSS
-    UINotifications::RssNotificationsChecker.new(options).deliver!
+    UINotifications::RssNotificationsChecker.new(options || {}).deliver!
```

Comment:
does this actually solve the issue? options is set to {} if its nil already?

I thought the real issue was at https://github.com/theforeman/foreman/blob/develop/app/services/ui_notifications/rss_notifications_checker.rb#L30
(note that options is an array vs hash)

Reply:
The issue is, that the `ActiveTask` passes options as nil. I could not understand why it is so, maybe the async task, as we discussed f2f.

Reply:
I think the error is normally not reproducible and @ohadlevy correct me if I'm wrong, but you ran this method manually from console. Normally the issue is probably not reproducible. But @ohadlevy  is right, the `[]` should be changed to `{}` on line he points to.

While this is most likely not normally reproducible since perform options default to `{}`, it's still valid bug.